h5py/tests/test_dtype.py

```diff
@@ -129,9 +129,14 @@ def test_vlen_enum(self):
 
 class TestEmptyVlen(TestCase):
     def test_write_empty_vlen(self):
+        try:
+            from numpy._core import records
```

Comment:
Is there a more public place to get this from?

Reply:
thanks. Done !

Reply:
thanks for asking, indeed there is ! `np.rec` is the public namespace for this, and is available in numpy 1 and 2 alike !